emloop/main_loop.py

```diff
@@ -184,7 +186,8 @@ def _run_epoch(self, stream: StreamWrapper, train: bool) -> None:
                 continue
             elif self._fixed_batch_size:
                 if batch_sizes != {self._fixed_batch_size}:
-                    var, len_ = [(k, len(v)) for k, v in batch_input.items() if len(v) != self._fixed_batch_size][0]
+                    var, len_ = [(k, len(v)) for k, v in batch_input.items()
```

Comment:
Please do not randomly reformat the code without changing its semantics

Reply:
@blazekadam I would love to, but the problem is that I have an automatic pep8 style checker that updates the whole file if I save it..

Reply:
Yea well you need to find some other way to check the code. Just see the update on line 198/199... :(